Add JUnit testsuite timeout in the Module Nature documentation (#404)
MicroEJ/docs: ApplicationDeveloperGuide/moduleNatures.rst

   * - microej.testsuite.cc.excludes.classes
     - Pattern of classes excluded from the code coverage analysis.
     - Not set
   * - microej.testsuite.timeout

>      - The time in seconds before any test is considerated as unknown. Set it to ``0`` to disable the time-out.

Reviewer: ```suggestion
     - The time in seconds before a test is considered as failed. Set it to ``0`` to disable the timeout.
```
